packages/api/src/utils/billing-types.ts

```diff
@@ -58,6 +59,23 @@ export interface CustomersService {
 
 export type StoragePriceName = 'free' | 'lite' | 'pro'
 
+export interface PriceTier {
+  flatAmount: number | null;
+  unitAmount: number | null;
+  upTo: number | null;
+}
+
+export interface CustomStoragePrice {
+  id: string;
+  bandwidth?: string;
+  label: string;
+  isPreferred: boolean;
```

Comment:
Will you add a comment explaining what this means?

Reply:
updated